SEED-platform/seed
# Refactor/organize filter and labels

#### Any background context you want to provide?
see ticket

#### What's this PR do?
refactors the beta inventory list filters and settings

#### How should this be manually tested?
try filtering on the table and note the new filter in the upper right (see screenshots for examples)

#### What are the relevant tickets?
https://github.com/SEED-platform/seed/issues/3073

#### Screenshots (if appropriate)
If available, filtering by a label:
<img width="1027" alt="Screen Shot 2022-01-06 at 3 52 09 PM" src="https://user-images.githubusercontent.com/564231/148469858-cd548247-3b4e-4044-ab4f-e578c412b81c.png">

Table filters show up in the filter list:
<img width="1024" alt="Screen Shot 2022-01-06 at 3 51 46 PM" src="https://user-images.githubusercontent.com/564231/148469885-c4e691b1-8a01-448f-91eb-5cda45d772a6.png">

Splits multiple column filters:
<img width="700" alt="Screen Shot 2022-01-06 at 5 04 04 PM" src="https://user-images.githubusercontent.com/564231/148469992-7c2bcd4b-916a-4cce-989a-060334c9ddec.png">

Deleting only part of a filter updates the column filter:
<img width="699" alt="Screen Shot 2022-01-06 at 5 04 15 PM" src="https://user-images.githubusercontent.com/564231/148470021-0eca3063-6a92-4535-a243-b702ebe73579.png">

Actions have been organized and converted to a dropdown menu:
<img width="471" alt="Screen Shot 2022-01-06 at 5 09 34 PM" src="https://user-images.githubusercontent.com/564231/148470090-75e60ec9-20ad-459c-a800-f89efd92f1d8.png">
